x-pack/test_serverless/api_integration/test_suites/security/cloud_security_posture/serverless_metering/cloud_security_metering.ts

```diff
@@ -0,0 +1,208 @@
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../../ftr_provider_context';
+
+import { RoleCredentials } from 'x-pack/test_serverless/shared/services';
+import {
+  deleteIndex,
+  addIndex,
+  createPackagePolicy,
+} from '../../../../../../test/api_integration/apis/cloud_security_posture/helper'; // eslint-disable-line @kbn/imports/no_boundary_crossing
+import {
+  cnvmFindingsMockDataForMetering,
+  cspmFindingsMockDataForMetering,
+  defendForContainersHeartbeatsForMetering,
+  kspmFindingsMockDataForMetering,
+} from './mock_data'; // eslint-disable-line @kbn/imports/no_boundary_crossing
+import {
+  LATEST_FINDINGS_INDEX_DEFAULT_NS,
+  LATEST_VULNERABILITIES_INDEX_DEFAULT_NS,
+} from '@kbn/cloud-security-posture-plugin/common/constants';
+import { UsageRecord, getInterceptedRequestPayload, setupMockServer } from './mock_usage_server';
+
+const CLOUD_DEFEND_HEARTBEAT_INDEX_DEFAULT_NS = 'metrics-cloud_defend.heartbeat-default';
+
+export default function (providerContext: FtrProviderContext) {
+  const mockUsageApiServer = setupMockServer();
+  const { getService } = providerContext;
+  const retry = getService('retry');
+  const kibanaServer = getService('kibanaServer');
+  const esArchiver = getService('esArchiver');
+  const es = getService('es');
+  const svlCommonApi = getService('svlCommonApi');
+  const svlUserManager = getService('svlUserManager');
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
+
+  /*
+  This test aims to intercept the usage API request sent by the metering background task manager.
+  The task manager is running by default in security serverless project in the background and sending usage API requests to the usage API.
+   This test mocks the usage API server and intercepts the usage API request sent by the metering background task manager.
+  */
+  describe('Intercept the usage API request sent by the metering background task manager', function () {
```

Comment:
will the test suit work on MKI? I'm assuming that as our other tests it will not, we probably should skip it right away with `skipMKI` tag

Reply:
You can find more information here https://docs.google.com/document/d/1tiax7xoDYwFXYZjRTgVKkVMjN-SQzBWk4yn1JY6Z5UY/edit#heading=h.iuivpo5zf9f7 ,  there is also an example of how to check if a test actually can be run on MKI, under `How to run tests in serverless QA` header. In short my understanding is that these tests are also a part of quality gates, not only do they run on Kibana CI. When they are part of quality gates, an actual serverless project is created in MKI env and tests run against this project

Reply:
Thanks for the clarification, I added the tag.